src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -289,7 +289,7 @@ public SignatureConstraint SignatureConstraint
 
                 return _signatureConstraint;
             }
-            protected set => _signatureConstraint = value;
+            protected init => _signatureConstraint = value;
```

Comment:
>init => _signatureConstraint = value [](http://example.com/codeflow?start=22&length=36)

Does this prevent this property from being called in the .ctor of derived classes? There are a few classes in Power Apps that use it.

Reply:
no :) 
We already have similar uses cases in Fx (If.cs, etc)  verifying that you can do that.